src/components/auth/ProfileModal/contents/notifications/PushNotificationContent.tsx

```diff
@@ -0,0 +1,162 @@
+import Button from '@/components/Button'
+import useSignMessage from '@/hooks/useSignMessage'
+import {
+  useCommitSignedMessageWithAction,
+  useGetFcmLinkingMessage,
+} from '@/services/api/notifications/mutation'
+import { getMessageToken } from '@/services/firebase/messaging'
+import { LocalStorage } from '@/utils/storage'
+import { sortObj } from 'jsonabc'
+import { useEffect, useState } from 'react'
+import { ContentProps } from '../../types'
+
+const STORAGE_KEY = 'push-notification-fcm-token'
```

Comment:
export this and make the name more clear
so you can use this in LogoutContent instead of duplicating the value

or maybe better, you can just export the storage instead of the storage key